db/seeds.rb

```diff
@@ -6,14 +6,35 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-employee1 = Employee.create(email: "ania@o2.pl", password: "lubiepomarancze")
-employee2 = Employee.create(email: "marek.z@gmail.com", password: "pass123456")
-employee3 = Employee.create(email: "ola.gola@hotmail.com", password: "piszjak123")
-employee4 = Employee.create(email: "pawel.tkaczyk@wp.pl", password: "hasloHASLO")
-employee5 = Employee.create(email: "zuza_gala@rv.com", password: "jak1to2jest")
+employee1 = Employee.where(email: 'ania@o2.pl').first_or_create!(password: 'password')
+employee2 = Employee.where(email: 'marek.z@gmail.com"').first_or_create!(password: 'password')
+employee3 = Employee.where(email: 'ola.gola@hotmail.com').first_or_create!(password: 'password')
+employee4 = Employee.where(email: 'pawel.tkaczyk@wp.pl"').first_or_create!(password: 'password')
+employee5 = Employee.where(email: 'zuza_gala@rv.com').first_or_create!(password: 'password')
 
-puts "Creating admin account"
-admin = AdminUser.create(email: "admin@example.com", password: 'tojestto123')
+admin = AdminUser.where(email: 'admin@example.com').first_or_create!(password: 'password')
 
-company_value1 = CompanyValue.create(title: "Patient")
-company_value2 = CompanyValue.create(title: "Helpful")
+company_value1 = CompanyValue.create!(title: "Patient")
+company_value2 = CompanyValue.create!(title: "Helpful")
+
+kudo1 = Kudo.create!(title: "At Kudos, we firmly believe that recognition is the best way to drive employee engagement and improve organizational performance. However, rewards can also be a terrific tool to enhance the employee experience.", content: "In practice, this recognition and communication first philosophy fosters connection, communication, and celebration of day-to-day contributions to significant accomplishments. We recommend that you keep the rewards component of your recognition program minimal or casual so they don't overshadow the program's goal. What people really want is to know they are valued, their work is noticed, and they belong. Rewards are nice but should not be the core focus. They should simply be a nice, low-cost benefit that reminds the team they work for a great company that cares about them.", employee_id: employee1.id, receiver_id: employee5.id, company_value: company_value1)
```

Comment:
Wow thats a lot to read! :) Consider using `Faker` gem which can take care of generating random names/sequences/quotes etc for you :)